script/binary-controller

```diff
@@ -42,7 +42,7 @@ else
 fi
 
 
-GIT_COMMIT=$(git describe --tags --dirty)
+GIT_COMMIT=$(git rev-parse --short HEAD)
```

Comment:
Let's add a clarifying comment here on why we are changing what upstream had.

Reply:
removed change here since I'm actually no longer building this binary using this script

Reply:
actually I am, I put it back